pms-migration-utility-default.properties

```diff
@@ -16,8 +16,12 @@ mosip.kernel.authmanager.url=http://authmanager.kernel
 
 spring.main.allow-circular-references=true
 service.token.request.clientId=mosip-pms-client
-service.token.request.issuerUrl=${mosip.api.internal.url}/v1/authmanager/authenticate/clientidsecretkey
-mosip.pms.utility.run.mode=upgrade
+service.token.request.issuerUrl=${mosip.kernel.authmanager.url}/v1/authmanager/authenticate/clientidsecretkey
+
+## allowed values for mosip.pms.utility.run.mode are 'upgrade' for migration and 'cronjob' for running it as cronjob. 
+mosip.pms.utility.run.mode=cronjob
```

Comment:
this being upgrade should be fine right ? why cronjob ?

Reply:
updated to upgrade